lib/src/util/__tests__/reactUtil.test.tsx

```diff
@@ -143,66 +199,98 @@ describe('reactUtil', function() {
     });
 
     it('wraps arrays that start with a string', function() {
-      const arr = ['bar', <span key="baz">baz</span>];
+      const arr = ['bar', <span key="baz">baz</span>],
+          result = ensureStartEndElements(arr) as ReactElement,
```

Comment:
Should this be `ensureElement`?
Same as a few below

Reply:
> Should this be ensureElement?
Same as a few below

Definitely.  Good catch. 423a4d257